props/additional_display_controllers.h

```diff
@@ -0,0 +1,398 @@
+       } break;
+      case EFFECT_TARGETTING:
+        if (img_.IMG_targetting)      // animation of targetting
+  ShowFileWithSoundLength(&img_.IMG_targetting,      jetpack_font_config.ProffieOSTargettingImageDuration);
+        else {
+  this->SetMessage("targetting");
+  this->SetScreenNow(SCREEN_MESSAGE);
+       } break;
+      case EFFECT_MISSILELAUNCH:
+        if (img_.IMG_missilelaunch)   // animation of jetpack launching missile
+  ShowFileWithSoundLength(&img_.IMG_missilelaunch,   jetpack_font_config.ProffieOSMissileLaunchImageDuration);
+        else {
+  this->SetMessage("launching\nmissile");
+  this->SetScreenNow(SCREEN_MESSAGE);
+       } break;
+      case EFFECT_MISSILEGOESBOOM:
+        if (img_.IMG_missilegoesboom) // animation of explosion
+  ShowFileWithSoundLength(&img_.IMG_missilegoesboom, jetpack_font_config.ProffieOSMissileGoesBoomImageDuration);
+        else {
+  this->SetMessage("missile\nexplodes");
+  this->SetScreenNow(SCREEN_MESSAGE);
+       } break;
+      case EFFECT_MANDOTALK:
+        if (img_.IMG_mandotalk)       // animation of mando & boba talking: "Nice shot! I was aiming for the other one!"
+  ShowFileWithSoundLength(&img_.IMG_mandotalk,       jetpack_font_config.ProffieOSMandoTalkImageDuration);
+        else {
+  this->SetMessage("nice shot");
+  this->SetScreenNow(SCREEN_MESSAGE);
+       } break;
+      case EFFECT_DISARM:
+        if (img_.IMG_disarm)          // animation of viewfinder going back up
+  ShowFileWithSoundLength(&img_.IMG_disarm,          jetpack_font_config.ProffieOSDisarmImageDuration);
+        else {
+  this->SetMessage("disarming");
+  this->SetScreenNow(SCREEN_MESSAGE);
+       } break;
+      // jetpack misshaps display effects:
+      // =================================
+      case EFFECT_FALSESTART:
+        if (img_.IMG_falsestart) // Jetpack with "smoke fart" animation
+  ShowFileWithSoundLength(&img_.IMG_falsestart,      jetpack_font_config.ProffieOSFalseStartImageDuration);
+        else {
+  this->SetMessage("jetpack\nfalse start");
+  this->SetScreenNow(SCREEN_MESSAGE);
+       } break;
+      case EFFECT_STUTTERING:
+        if (img_.IMG_stuttering) // Jetpack with trail of "smoke farts" animation
+  ShowFileWithSoundLength(&img_.IMG_stuttering,      jetpack_font_config.ProffieOSStutteringImageDuration);
+        else {
+  this->SetMessage("jetpack\nstuttering");
+  this->SetScreenNow(SCREEN_MESSAGE);
+       } break;
+      case EFFECT_SELFDESTRUCT:
+        if (img_.selfdestruct) // Jetpack exploding animation
+  ShowFileWithSoundLength(&img_.selfdestruct,        jetpack_font_config.ProffieOSSelfDestructImageDuration);
+        else {
+  this->SetMessage("jetpack\nself destruct");
+  this->SetScreenNow(SCREEN_MESSAGE);
+       } break;
+      case EFFECT_MELTDOWN:
+        if (img_.meltdown) // Jetpack melting animation
+  ShowFileWithSoundLength(&img_.meltdown,            jetpack_font_config.ProffieOSMeltdownImageDuration);
+        else {

```

Comment:
It should always be "} else {" not "else {" or "} else".

Reply:
If both sides are short one-liners, then "else" (no brackets) is also acceptable.